pkg/didcomm/protocol/didexchange/states_test.go

```diff
@@ -51,69 +52,69 @@ func TestNoopState(t *testing.T) {
 
 // null state can transition to invited state or requested state.
 func TestNullState(t *testing.T) {
-	null := &null{}
-	require.Equal(t, "null", null.Name())
-	require.False(t, null.CanTransitionTo(null))
-	require.True(t, null.CanTransitionTo(&invited{}))
-	require.True(t, null.CanTransitionTo(&requested{}))
-	require.False(t, null.CanTransitionTo(&responded{}))
-	require.False(t, null.CanTransitionTo(&completed{}))
+	nul := &null{}
```

Comment:
accidental refactors?

Reply:
no.. wanted to remove `shadowing warning`